test/Tests/src/Network/Http_Spec.enso

```diff
@@ -1,332 +1,205 @@
 from Standard.Base import all
-import Standard.Base.Errors.Common.Syntax_Error
 
-import Standard.Base.Network.HTTP.Form.Form
-import Standard.Base.Network.HTTP.Request.Request
+import Standard.Base.Errors.Common.Forbidden_Operation
+import Standard.Base.Errors.Common.Syntax_Error
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
 import Standard.Base.Network.HTTP.Request_Body.Request_Body
 import Standard.Base.Network.HTTP.Request_Error
 import Standard.Base.Network.Proxy.Proxy
+import Standard.Base.Runtime.Context
 
 from Standard.Test import Test, Test_Suite
 import Standard.Test.Extensions
 
 polyglot java import java.lang.System as Java_System
 
+type Test_Type
+    Aaa (s:Text)
+    Bbb (i:Integer)
+
+type Bad_To_Json
+    Aaa (s:Text)
+    to_json self = JS_Object.from_pairs [Pair.new [] 0]
+
 spec =
+    # Set this to "https://httpbin.org" in your env to run this test
     base_url = Java_System.getenv "ENSO_HTTP_TEST_HTTPBIN_URL"
-    pending = if base_url != Nothing then Nothing else
-        "The HTTP tests only run when the `ENSO_HTTP_TEST_HTTPBIN_URL` environment variable is set to URL of the httpbin server"
+    pending_has_url = if base_url != Nothing then Nothing else
+        "The POST tests only run when the `ENSO_HTTP_TEST_HTTPBIN_URL` environment variable is set to URL of the httpbin server"
```

Comment:
I think this suite is not only running POST tests. Why the name change here?

Reply:
Done